src/news/2021-05-08-Inverted-Wordles.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Inverted Wordles
+filename: Inverted-Wordles
+date: '2021-05-08'
+---
+
+In a traditional word cloud, the most popular answers (e.g. words, phrases) are emphasized. The result of this emphasis
+is for the minority to fade away into the background. With their work on Inverted Wordles, the We Count team is seaking
+a way to invert that relationship and provide a means to amplify those minority voices.
+
+<figure>
+    <img src="images/wordle.png"
+        alt="A screenshot showing a wordle generated for the question,
+        What are the three most important problems to be addressed by inclusive design?">
+    <figcaption>
+        <details>
+            <summary>
+                A screenshot showing a wordle generated for the question, "What are the three most important problems to
+                be addressed by inclusive design?"
+            </summary>
+            Answers include:
+            <ul>
+                <li>hope</li>
+                <li>engagement</li>
+                <li>power</li>
+                <li>empathy</li>
+                <li>co-design</li>
+                <li>captitalism</li>
+                <li>needs</li>
+                <li>shelter</li>
+                <li>barriers</li>
+                <li>outliers</li>
+                <li>inflexibility</li>
+                <li>communication</li>
+                <li>margins</li>
+                <li>etc.</li>
+            </ul>
```

Comment:
I don't think this is the sort of detail that is interesting. For someone who can not perceive the image, what details are relevant?

The size, shape, arrangement of the words have (in)significance.

Something like this could work:
"The image shows different answers to the question 'What are the three most important problems to be addressed by inclusive design?'. As this is an inverted wordle, the least popular answers appear larger, and the more popular responses appear smaller. For example the words 'capitalism' and 'inflexibility' appear more prominently, while words like 'community' and 'comraderie' appear smaller. Visually all these words are arranged at different angles and appear in different colours. The angles and colours of words do not have any significance in this example."

Reply:
@cindyli could you comment on if this is an accurate description of the inverted world image. It's the same one that is on https://wecount.inclusivedesign.ca/views/inverted-wordles/